bioptim/optimization/solution.py

```diff
@@ -1029,5 +1048,6 @@ def print_constraints(ocp, sol):
             print(f"Solving time: {self.time_to_optimize} sec\nElapsed time: {self.real_time_to_optimize} sec")
             self.print(CostType.OBJECTIVES)
             self.print(CostType.CONSTRAINTS)
+            print_parameters(self.ocp, self)
```

Comment:
Please add a new CostType.PARAMETERS so it can be printed alone as the other types